tasks/managed/push-snapshot/push-snapshot.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       description: The workspace where the snapshot spec and data json files reside
   steps:
     - name: push-snapshot
-      image: quay.io/konflux-ci/release-service-utils:e85ceb962ee6f4d0672b4aa4e9946621ab302f20
+      image: quay.io/konflux-ci/release-service-utils:change_me
```

Comment:
This will be changed when https://github.com/konflux-ci/oras-container/pull/172 is merged and a new utils image is built and the mint both updates the utils image.